buffer/src/main/java/io/netty/buffer/SwappedByteBuf.java

```diff
@@ -36,7 +36,7 @@
 @Deprecated
 public class SwappedByteBuf extends ByteBuf {
 
-    private final ByteBuf buf;
+    final ByteBuf buf;
```

Comment:
also add a todo here to make private again once https://github.com/netty/netty/pull/6081 is fixed.